common/djangoapps/student/cookies.py

```diff
@@ -5,6 +5,8 @@
 import time
 import json
 
+from email_marketing.cookies import add_email_marketing_cookies
```

Comment:
I'm looking at this, and realizing I hadn't noticed before the somewhat awkward dependency that this adds from `common/djangoapps/student` to `lms/djangoapps/email_marketing`. I'm going to think a little on whether there's a better way to manage this so that we can avoid this dependency.